### DISCUSSION TITLE
Is there any option to shift+click

### DISCUSSION THREAD

**Discussion by lakshminarayanagrahari-tomtomexternal** [getgauge/taiko#2604] [General]
Hi,
I am new to gauge taiko. I am working on an automation script . I want to simulate shift + click. Basically the requirement is to press shift and clicking different element on screen while holding shift key.

Please suggest me how can I do that. 

Regards

**Comment by DCoomer** [comment]
try await click(element, {modifiers: 8});

The ability to use modifiers from https://chromedevtools.github.io/devtools-protocol/tot/Input/#method-dispatchMouseEvent
was added recently from: [2587](https://github.com/getgauge/taiko/pull/2592)

**Reply by ghost** [reply]
Intresting thread! Offtop, possible simulate with using mouse moving with always pressed button (button shift pressed when mouse moving) ?

**Reply by DCoomer** [reply]
No idea, dragAndDrop() with the click modifier might work. Or you might have to call the CDP methods directly

**Comment by lakshminarayanagrahari-tomtomexternal** [comment]
@DCoomer, it is working. Thanks a lot.
